articles/machine_learning_engineering/experimental_trackers/Diffusion_Model_Training_Gradio.ipynb

```diff
@@ -0,0 +1,9492 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        global generated_grid_path
</pre><p>Let's pass this variable as a parameter to the evaluate function. By modifying global variables you can introduce wacky bugs that are hard to debug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/decodingml/articles-code/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>